interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -107,6 +107,11 @@ namespace cling {
     }
 
     NodeContext VisitStmt(Stmt* S) {
+      for (auto I = S->child_begin(), E = S->child_end(); I != E; ++I) {
+        if (*I)
+          NodeContext nc = Visit(*I);
+      }
+
       return NodeContext(S);
     }
 
```

Comment:
...and then `Visit()` can return `void` (see the template arg of the `RecursiveXYZVisitor`